tests/slack_sdk/models/test_views.py

```diff
@@ -30,6 +30,7 @@ def setUp(self) -> None:
     def verify_loaded_view_object(self, file):
         input = json.load(file)
         view = View(**input)
+        self.assertTrue(view.state is None or isinstance(view.state, ViewState))
```

Comment:
This assertion used to be failing before the change. Before the change in this PR, the passing one is `self.assertTrue(view.state is None or isinstance(view.state, dict))`.